src/lib389/lib389/cli_conf/plugins/rootdn_ac.py

```diff
@@ -96,39 +96,39 @@ def rootdn_edit(inst, basedn, log, args):
 def _add_parser_args(parser):
     parser.add_argument('--allow-host', nargs='+',
                         help='Sets what hosts, by fully-qualified domain name, the root user is allowed to use '
-                             'to access the Directory Server. Any hosts not listed are implicitly denied '
+                             'to access Directory Server. Any hosts not listed are implicitly denied '
                              '(rootdn-allow-host)')
     parser.add_argument('--deny-host', nargs='+',
                         help='Sets what hosts, by fully-qualified domain name, the root user is not allowed to use '
-                             'to access the Directory Server Any hosts not listed are implicitly allowed '
+                             'to access Directory Server. Any hosts not listed are implicitly allowed '
                              '(rootdn-deny-host). If an host address is listed in both the rootdn-allow-host and '
```

Comment:
I think it should be 'a host' instead of 'an host'

Reply:
Good catch. I missed that. The same typo is in src/cockpit/389-console/src/lib/plugins/rootDNAccessControl.jsx.
I fixed both occurrences.